src/utils/save_game/save_game.cpp

```diff
@@ -0,0 +1,66 @@
+#include "save_game.h"
+#include "../../creature/creature.h"
+#include "../../creature/main_character/main_character.h"
+#include <iostream>
+#include <fstream>
+
+void SaveGame(MainCharacter& main_characher, Creature& creature, const std::string& filename)
```

Comment:
Why would we need to save the stats of a single creature?

Also, why is the name of the file a parameter here? I believe that it should be defined inside this function which is solely responsible for creating/modifying it.

Reply:
should i take the creature out, or add the ability for more creatures to be saved?

Reply:
I'd say take it out. We can always add it later if we need it. For now, there is no need for it.